css/public/style.scss

```diff
@@ -1,841 +0,0 @@
-.contactdetails__header .contactdetails__title::-moz-placeholder { /* Mozilla Firefox 19+ */
-	color: #fff; /* No vars used on purpose since we use custom BGs */
-	opacity: .8;
-}
-.contactdetails__header .contactdetails__name:-ms-input-placeholder,
-.contactdetails__header .contactdetails__org:-ms-input-placeholder,
-.contactdetails__header .contactdetails__title:-ms-input-placeholder { /* Internet Explorer 10-11 */
-	color: #fff; /* No vars used on purpose since we use custom BGs */
-	opacity: .8;
-}
-.contactdetails__header .contactdetails__name:placeholder-shown,
-.contactdetails__header .contactdetails__org:placeholder-shown,
-.contactdetails__header .contactdetails__title:placeholder-shown { /* Standard (https://drafts.csswg.org/selectors-4/#placeholder) */
-	color: #fff; /* No vars used on purpose since we use custom BGs */
-	opacity: .8;
-}
-
-.contactdetails__header #details-actions a,
-.contactdetails__header #details-actions button {
-	padding: 15px;
-	background-color: transparent;
-	border: none;
-	opacity: .5;
-	margin: 3px;
-}
-.contactdetails__header #details-actions a:hover,
-.contactdetails__header #details-actions button:hover,
-.contactdetails__header #details-actions a:focus,
-.contactdetails__header #details-actions button:focus {
-	opacity: .7
-}
-
-avatar {
-	position: relative;
-	height: 75px;
-	width: 75px;
-	border-radius: 50%;
-	overflow: hidden;
-	flex-shrink: 0;
-}
-
-.contactdetails__logo {
-	height: 75px;
-	width: 75px;
-	object-fit: cover;
-	background-color: transparent;
-	margin: 0;
-	cursor: pointer;
-}
-
-.avatar-options {
-	top: 0;
-	background-color: rgba(0,0,0,0.2);
-	display: none;
-	width: 100%;
-	height: 100%;
-	overflow: hidden;
-}
-.avatar-options > [class^="icon-"] {
-	display: none;
-	opacity: 0.5;
-	cursor: pointer;
-	min-width: 20px;
-	min-height: 20px;
-}
-.avatar-options:hover > [class^="icon-"] {
-	opacity: 0.6;
-}
-.avatar-options > [class^="icon-"]:hover {
-	opacity: 0.8;
-}
-
-avatar.avatar--missing .avatar-options {
-	display: flex;
-	border-radius: 50%;
-}
-avatar:not(.maximized).avatar--missing .avatar-options .icon-upload-white {
-	display: block;
-	width: 100%;
-	height: 100%;
-}
-
-avatar.maximized {
-	position: fixed;
-	height: 100%;
-	width: 100%;
-	top: 0;
-	left: 0;
-	border-radius: 0;
-	padding: 20px;
-	background-color: rgba(0, 0, 0, 0.9);
-	display: flex;
-	justify-content: center;
-	z-index: 200;
-	padding-top: 65px; /* Nextcloud header */
-}
-avatar.maximized img {
-	width: auto;
-	height: auto;
-	border-radius: 0;
-	max-height: calc(100% - 40px);
-	max-width: 100%;
-	align-self: center;
-	margin-bottom: 40px;
-}
-avatar.maximized .avatar-options {
-	height: 40px;
-	position: absolute;
-	top: calc(100% - 40px);
-	left: 0;
-	display: flex;
-	opacity: 1;
-	padding: 0;
-	flex-wrap: wrap;
-	justify-content: space-around;
-}
-avatar.maximized .avatar-options > [class^="icon-"] {
-	min-width: 25%;
-	display: block;
-}
-
-#check-all + label:before {
```

Comment:
This is the only rule I removed. I wasn't able to find any place it was used (it was added by me in one of the initial commits)